src/utils/analytics.ts

```diff
@@ -1,13 +1,21 @@
 // Make sure we're in the browser
+import { type Session } from "next-auth";
 import posthog from "posthog-js";
 import { env } from "~/env.mjs";
 
 const enableAnalytics = typeof window !== "undefined";
-
 if (enableAnalytics) {
   if (env.NEXT_PUBLIC_POSTHOG_KEY) {
     posthog.init(env.NEXT_PUBLIC_POSTHOG_KEY, {
-      api_host: "https://app.posthog.com",
+      api_host: `${env.NEXT_PUBLIC_HOST}/ingest`,
     });
   }
 }
+
+export const identifySession = (session: Session) => {
+  if (!enableAnalytics) return;
+  posthog.identify(session.user?.id, {
```

Comment:
What happens if `user` is null? Should we check for that and not call `identify` in that case?

Reply:
Makes sense